### ISSUE TITLE
Questions about new code deployed on production 

### ISSUE THREAD

**Issue by libzhang** [openshift/secondary-scheduler-operator#76]
Hi Experts, 
I have some questions need your help: 
1. is there any guide for CLI installation on production environment ?  I see we have UI method to install via operatorHub. 
2. when will the latest operator image published to operatorHub?  can we build ourselves and update the operator image in operatorHub ? 
3. if we must install operator via operatorHub in production enviroment,  then any way to skip the UI manually operation, we want it to be automated.

**Comment by ingvagabund** [comment]
> then for the manifests https://github.com/openshift/secondary-scheduler-operator/tree/master/deploy, from my understanding, it did not have a OLM(Operator Lifecylce Manager) support, correct ?

Correct

> for the second way, https://github.com/openshift/secondary-scheduler-operator#operatorhub-install-with-custom-index-image, I am not quite understand it, does it mean after the related steps, in operatorHub, there will be a custom operator image I built ?

The steps will only produce the index image. Sharing the operand on the operatorHub requires additional steps. I don't have the right steps to do it. In my environment, creating an index image and deploying a CatalogSource resource was sufficient. OpenShift comes with default installation of the OLM and operator catalog which picks the deployed CatalogSource as another source for the in-cluster running operator catalog.

**Comment by libzhang** [comment]
Hi @ingvagabund, thanks a lot for your support.  for the second way, I will have a try.  and maybe still some questions, let's have this thread open.  appreciate your help again.

**Comment by ingvagabund** [comment]
Happy to help. If anything, we can extend the current readme with additional steps that can help in the future.

**Comment by libzhang** [comment]
Hi @ingvagabund,

**Comment by libzhang** [comment]
>build an image index (https://github.com/openshift/secondary-scheduler-operator#operatorhub-install-with-custom-index-image) and use CLI to install the operator.

Hi @ingvagabund 
regarding this, I can build an index image now. what else needs to do to install the secondary scheduler operator via CLI? 
create a CatalogSource and then the OLM will automatically do the deployment?

**Comment by ingvagabund** [comment]
I am unfortunately missing knowledge in the next steps after the CatalogSource is deployed. One still needs to create the namespace and an installplan resource I suppose. I suggest checking the OLM documentation for further pointers.

**Comment by libzhang** [comment]
Hi @ingvagabund, 
I managed to deploy the official secondary scheduler operator via CLI. It needs an OperatorGroup and a Subscription.
my questions, 
1. when will redhat release a new operator image to the catalog source, readhat-operators ?  we want to use the latest version. 
   In our system, csv version is v1.1.0, which is built at Jun 12, 2022.   The recent fix in this repo is critical for us to use. 
2. from my understanding, If I create of my own index image and catalog resource, I need to maintain it myself and it is actually the same work currently in "OperatorHub"  ---> "openshift-secondary-scheduler-operator",  so create our-own for the same operator seems  not necessary.  we just need a new version that fix the recent bugs and published to the redhat-operator catalog,  correct ? 
3. when I built the index image from scratch, I met an error, 
```
$ docker build -t my-custom-registry/secondary-scheduler-operator-metadata:${IMAGE_TAG} -f Dockerfile.metadata .
Sending build context to Docker daemon  63.89MB
Step 1/7 : FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
unauthorized: authentication required
```
